pkg/internal/checkup/clountinitconfig/configfiles.go

```diff
@@ -25,11 +25,47 @@ import (
 	"github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/config"
 )
 
+const (
+	CfgScriptName = "dpdk-checkup-boot.sh"
+	BinDirectory  = "/usr/bin/"
+)
+
 type Config struct {
+	isolatedCores string
 }
 
 func NewConfig() Config {
+	const isolatedCores = "2-7"
 
 	return Config{
+		isolatedCores: isolatedCores,
 	}
 }
+
+func (c Config) GenerateCfgFile() string {
+	const cloudInitScriptTemplate = `#!/bin/bash
+set -e
+
+checkup_tuned_adm_set_marker_full_path=%s
+
+driverctl set-override %s vfio-pci
+driverctl set-override %s vfio-pci
+
+mkdir -p /mnt/huge
+mount /mnt/huge --source nodev -t hugetlbfs -o pagesize=1GB
```

Comment:
This is no longer needed since #218 got merged.

Reply:
yes it conflicts. will fix

Reply:
DONE